bin/plotting/pycbc_plot_singles_vs_params

```diff
@@ -148,7 +149,7 @@ hexbin_style = {
 
 # In earlier versions mpl will try to take the max over bins with 0 triggers
 # and fail, unless we tell it to leave these blank by setting mincnt
-if matplotlib.__version__ < '3.8.1':
+if Version(matplotlib.__version__) < Version('3.8.1'):
     hexbin_style['mincnt'] = 0
```

Comment:
Trying to remember things from a decade ago, I *think* with newer matplotlibs we should actually set mincnt to 1. I would still want empty bins to show up as blanks, because I want to be able to tell the difference between empty bins and bins containing only a few triggers.

Reply:
mincnt will be `None` for newer versions, and mincnt is then forced to be 1 [here](https://github.com/matplotlib/matplotlib/blob/a3682f32ef19561b790fa4d8639a8eadae6ffbc1/lib/matplotlib/axes/_axes.py#L5286)